user-guide/Reference/Skyline_DataMiner_Folder/Overview_of_the_different_subfolders.md

```diff
@@ -33,4 +33,6 @@ uid: Overview_of_the_different_subfolders
 | Webpages | All webpages of the DataMiner client applications. | No |
 
 > [!NOTE]
-> Files marked as "synchronized" in the rightmost column are synchronized among all DMAs in the cluster every day at midnight (00:00). Therefore, it is of great importance that the time settings of all DMAs in the cluster are synchronized as well. For more information, see [How do I synchronize time settings within a DMS?](xref:General_configuration#how-do-i-synchronize-time-settings-within-a-dms)
+>
+> - Files marked as "synchronized" in the rightmost column are synchronized among all DMAs in the cluster every day at midnight (00:00). Therefore, it is of great importance that the time settings of all DMAs in the cluster are synchronized as well. For more information, see [How do I synchronize time settings within a DMS?](xref:General_configuration#how-do-i-synchronize-time-settings-within-a-dms)
+> - The *Services* folder contains the services hosted on the DataMiner Agent. For services hosted on other DMAs, the folder `C:\Skyline DataMiner\RemoteServices` contains subfolders with the DMA ID, and this is where they should be synchronized.
```

Comment:
What do you mean with "this is where they should be synchronized"? Do you mean that the service info in the subfolders is synchronized with the info in the Services folder on the DMAs with the corresponding IDs?